transformation/plugins/org.jboss.tools.fuse.transformation.editor/src/main/java/org/jboss/tools/fuse/transformation/editor/internal/wizards/Hl7Page.java

```diff
@@ -117,7 +118,7 @@ public void handleValueChange(ValueChangeEvent event) {
                 }
                 if (path != null) {
                     try {
-                        IResource resource = model.getProject().findMember(path);
+                        IResource resource = CamelUtils.project().findMember(path);
```

Comment:
Seems cleaner to be based on the provided model.getProject(). Relying on CamelUtils.Project() introduce some implicit dependency of an Editor already opened and that the one with focus is the correct one.